app-admin/systemrescuecd-x86/systemrescuecd-x86-4.6.0.ebuild

```diff
@@ -18,8 +18,14 @@ S=${WORKDIR}
 RESTRICT="mirror"
 
 src_install() {
-	insinto "/usr/share/${PN%-*}"
-	doins "${DISTDIR}/${P}.iso"
+	insinto /usr/share/${PN%-*}
+	doins ${DISTDIR}/${P}.iso
+
+	exeinto /etc/grub.d
+	newexe ${FILESDIR}/systemrescuecd.grub 39_systemrescuecd
```

Comment:
Now here's a big issue: this will collide when multiple versions of sysresccd are installed. I kinda think we need to move those files to a separate package. It would be perfect if you could assemble a common `systemrescuecd-grub` that finds all systemrescuecd versions (including possibly non-x86) and handles grub entries for them.